src/fides/api/alembic/migrations/versions/a6d9cdfcc7dc_migrate_remaining_data_categories.py

```diff
@@ -0,0 +1,99 @@
+    remove_conflicting_rule_targets,
+    update_ctl_policies,
+    update_data_label_tables,
+    update_datasets_data_categories,
+    update_default_dsr_policies,
+    update_privacy_declarations,
+    update_rule_targets,
+    update_system_ingress_egress_data_categories,
+)
+
+# revision identifiers, used by Alembic.
+revision = "a6d9cdfcc7dc"
+down_revision = "f712aa9429f4"
+branch_labels = None
+depends_on = None
+
+###############
+## Data Uses ##
+###############
+"""
+The `key` is the old value, the `value` is the new value
+These are ordered specifically so that string replacement works on both parent and child items
+"""
+data_use_upgrades: Dict[str, str] = {}
+
+#####################
+## Data Categories ##
+#####################
+"""
+The `key` is the old value, the `value` is the new value
+These are ordered specifically so that string replacement works on both parent and child items
+"""
+data_category_upgrades: Dict[str, str] = {
+    "user.biometric_health": "user.biometric.health",
+    "user.credentials.biometric_credentials": "user.authorization.biometric",
+    "user.credentials.password": "user.authorization.password",
+}
+
+
+def upgrade() -> None:
+    """
+    Given that our advice is to turn off auto-migrations and make a db copy,
+    there is no "downgrade" version of this. It also wouldn't be feasible given
+    it would require an older version of fideslang.
+    """
+    bind: Connection = op.get_bind()
+
+    logger.info("Removing old default data categories")
+    bind.execute(text("DELETE FROM ctl_data_categories WHERE is_default = TRUE;"))
+
+    logger.info("Upgrading additional Privacy Declarations for Fideslang 2.0")
+    update_privacy_declarations(bind, data_use_upgrades, data_category_upgrades)
+
+    logger.info("Upgrading additional Policy Rules for Fideslang 2.0")
+    update_ctl_policies(bind, data_use_upgrades, data_category_upgrades)
+
+    logger.info("Upgrading additional Data Categories in Datasets")
+    update_datasets_data_categories(bind, data_category_upgrades)
+
+    logger.info("Upgrading additional Data Categories in System egress/ingress")
+    update_system_ingress_egress_data_categories(bind, data_category_upgrades)
+
+    logger.info("Updating additional Rule Targets")
+    update_rule_targets(bind, data_category_upgrades)
+
+    logger.info("Upgrading additional Taxonomy Items for Fideslang 2.0")
+    update_data_label_tables(bind, data_category_upgrades, "ctl_data_categories")
+
+    session = Session(bind=bind)
+    # insert new rule targets directly into the database for the default policies
+    update_default_dsr_policies(session)
+    # remove conflicting rule targets from all erasure policies
+    remove_conflicting_rule_targets(session)
+
+
+def downgrade() -> None:
+    """
+    This migration does not support downgrades.
+    """
+    logger.info(
+        "Data migrations from Fideslang 2.0 to Fideslang 1.0 are not supported."
```

Comment:
Maybe update the message to differ from where it was copied in case this gets hit to not cause confusion in tracking it down.